lib/ui/key.dart

```diff
@@ -14,7 +14,49 @@ enum KeyEventType {
   up,
 
   /// The key is held, causing a repeated key input.
-  repeat,
+  repeat;
+
+  String get label {
+    return switch (this) {
+      down => 'Key Down',
+      up => 'Key Up',
+      repeat => 'Key Repeat',
+    };
+  }
+}
+
+/// The source device for the key event.
+///
+/// Not all platforms supply an accurate type.
+///
+/// Defaults to [keyboard].
```

Comment:
I think default value should belong to a variable/field, not a type.

Reply:
Yes, good point.  I already removed this comment in other places, I just missed this one.